samples/net/dns_resolve/README.rst

```diff
@@ -17,6 +17,10 @@ The multicast DNS (mDNS) client resolver support can be enabled by setting
 CONFIG_MDNS_RESOLVER Kconfig variable.
 See https://tools.ietf.org/html/rfc6762 for more details about mDNS.
 
+The link-local multicast name resolution (LLMNR) client resolver support can be
+enabled by setting CONFIG_LLMNR_RESOLVER Kconfig variable.
```

Comment:
change to this (to get a cross reference to the config option documentation):

    enabled by setting the :option:`CONFIG_LLMNR_RESOLVER` Kconfig option.